src/plugins/files/server/mocks.ts

```diff
@@ -56,7 +57,9 @@ export const createFileMock = (): DeeplyMockedKeys<File> => {
     share: jest.fn(),
     listShares: jest.fn(),
     unshare: jest.fn(),
-    toJSON: jest.fn(),
+    toJSON: jest.fn(() => {
+      return clone(fileMock.data);
```

Comment:
Just out of curiosity: why clone?

Reply:
I used `clone()` so that the data being return by this mock is not a reference to the data stored in the instance. Thus, the data returned can be Mutated (in a tests) without impacting the state of the `File` class instance